docs/index.rst

```diff
@@ -595,6 +678,50 @@ Users
        "last_name": "Watson"
      }
 
+.. http:post:: /password-resets/
```

Comment:
Ah, I see. I'm a bit more in favor of the current approach tbh, happy to explain my reasoning if you'd like to hear. For now, going to go with what we have at the moment.